### PR TITLE
Promote emptyDir wrapper volumes tests to Conformance

### PR DESCRIPTION
**Pull request by mgdevstack**
**What this PR does / why we need it**:
Promoting emptyDir wrapper volume tests for Conformance.
> 1. [sig-storage] EmptyDir wrapper volumes should not conflict
>2. [sig-storage] EmptyDir wrapper volumes should not cause race condition when used for configmaps [Serial] [Slow]

This test is NOT being promoted to Conformance:
>3. [sig-storage] EmptyDir wrapper volumes should not cause race condition when used for git_repo [Serial] [Slow]

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #
Reference: https://github.com/kubernetes/kubernetes/pull/68023#issuecomment-418833594. Though 2 tests are [SLOW] but can be considered for Conformance promotion.
 
**Special notes for your reviewer**:
- No Flakyness
- Following are [SLOW] tests -
1. `should not cause race condition when used for configmaps` [~180 Seconds]
2. `should not cause race condition when used for git_repo` [~150 Seconds]
- Improving API covered at some extent for `GET, DELETE /api/v1/namespaces/{namespace}/persistentvolumeclaims`.

**Release note**:

```release-note
NONE
```
/area conformance
@kubernetes/sig-storage-pr-reviews

### CODE REVIEW

File: test/e2e/storage/empty_dir_wrapper.go [hunk 1]
```diff
@@ -181,7 +191,13 @@ var _ = utils.SIGDescribe("EmptyDir wrapper volumes", func() {
...
 	})
 
-	It("should not cause race condition when used for git_repo [Serial] [Slow]", func() {
+	// SLow by design [~150 Seconds]
+	/*
+		Release : v1.13
+		Testname: EmptyDir Wrapper Volume, GitRepo volumes, no race
+		Description: Create a Git Server service exposing URL and repo. 50 GitRepo Volumes are created with this Git Repository. 5 replicas of pod is created mounting GitRepo Volumes. Pod MUST NOT fail waiting for Volumes.
+	*/
+	framework.ConformanceIt("should not cause race condition when used for git_repo [Serial] [Slow]", func() {
```
Comment (msau42) [inline test/e2e/storage/empty_dir_wrapper.go]: I don't think we should make a conformance test for gitRepo, which is deprecated

Comment (msau42) [reply test/e2e/storage/empty_dir_wrapper.go]: Oh actually i think the test was changed to not use the git repo volume.  We should update the test name and description to reflect that

Comment (msau42) [reply test/e2e/storage/empty_dir_wrapper.go]: Thanks for summarizing! I think what you said makes sense.  Let's drop the git_repo race test from conformance, but promote the other 2.  Maybe you can also add a comment to the git_repo test explaining why we can't promote

File: test/e2e/storage/empty_dir_wrapper.go [hunk 2]
```diff
@@ -172,7 +176,13 @@ var _ = utils.SIGDescribe("EmptyDir wrapper volumes", func() {
 	// but these cases are harder because tmpfs-based emptyDir
 	// appears to be less prone to the race problem.
 
-	It("should not cause race condition when used for configmaps [Serial] [Slow]", func() {
+	// SLow by design [~180 Seconds]
```
Comment (spiffxp) [inline test/e2e/storage/empty_dir_wrapper.go]: Let's put this in the Description field instead

Comment (mgdevstack) [reply test/e2e/storage/empty_dir_wrapper.go]: Done.

File: test/e2e/storage/empty_dir_wrapper.go (hunk 1 above)
Comment (mgdevstack) [reply test/e2e/storage/empty_dir_wrapper.go]: Done.

**Comment by spiffxp** [comment]
/kind cleanup
/assign @bgrant0607 @smarterclayton 
for the conformance /approve

**Comment by bgrant0607** [comment]
The test name "should not conflict" could be a lot more specific, but that could be fixed in a follow-up PR.

Good catch on the gitRepo test. I updated the PR description.

/approve

**Pull request merged**